### DISCUSSION TITLE
Getting non object when using useIsMutating

### DISCUSSION THREAD

**Discussion by cubecleveland** [TanStack/query#2572] [Q&A]
Hellow, im trying to use the useIsMutating hook.

NOt doing anything but importing at the top of the file
import { useIsMutating } from "react-query";

then using 
 const isMutatingDoor = useIsMutating();

Gettting an Error thrown with a non object type..... 

am i doing anything woring.... is that still supported ?

**Comment by TkDodo** [comment]
Is this the same as: https://github.com/tannerlinsley/react-query/discussions/2535 ?

**Comment by cubecleveland** [comment]
Yes. resolved. my apologies for not searching before.
